src/led.h

```diff
@@ -26,8 +26,19 @@
 #ifndef LED_H
 #define LED_H
 
+#define NEWLED 1
```

Comment:
Should probably add a comment describing what this `NEWLED` define does? :shrug:

Reply:
Apologies. I was being very careful not to expunge the older LED handling, in case there was some rationale in it which had escaped me.